src/core/withExperiment.js

```diff
@@ -0,0 +1,76 @@
+/* @flow */
+import * as React from 'react';
+import invariant from 'invariant';
+import cookie from 'react-cookie';
+
+export type ExposedWithExperimentProps = {|
+  variant: string,
+|};
+
+type Props = {|
+  cookieConfig?: Object,
+  id: string,
+  variantA: string,
+  variantB: string,
+|};
+
+type InternalProps = {|
```

Comment:
I noticed you renamed this but I wasn't sure why. Anyway, so I left for now but wanted to double check with you on this since we usually keep this name or let me know if I am missing something

Reply:
I renamed this type to be more explicit. `InternalProps` is nice when it is in a "component" file, with only a component definition (class or function) in it. In this file, there is mainly a HOC (that uses a component class).